scripts/run-docker.sh

```diff
@@ -2,10 +2,12 @@
 set -e -u
 
 HOME=/home/builder
+REPOROOT="$(dirname $(readlink -f $0))/../"
+
 IMAGE_NAME=termux/package-builder
 CONTAINER_NAME=termux-package-builder
 
-[ `id -u` -eq 0 ] && USER=root || USER=builder
+[ $(id -u) -eq 0 ] && USER=root || USER=builder
```

Comment:
@hacknox @rnauber This causes `sudo ./scripts/run-docker.sh` to login as root instead of as builder. Is this line really necessary for getting rid of the permission errors?